kong/client.go

```diff
@@ -4,17 +4,22 @@ import (
 	"context"
 	"encoding/json"
 	"io"
+	"net"
 	"net/http"
 	"net/http/httputil"
 	"net/url"
 	"os"
+	"time"
 
 	"github.com/pkg/errors"
 
 	"github.com/kong/go-kong/kong/custom"
 )
 
-const defaultBaseURL = "http://localhost:8001"
+const (
+	defaultBaseURL = "http://localhost:8001"
+	defaultTimeout = 60 * time.Second
```

Comment:
Consider exporting the default timeout and adding a godoc comment, for sake of self-documenting code.

This will make a difference for readers of autogenerated docs.

Reply:
Done